### PR TITLE
Combiner lifting

### PR DESCRIPTION
**Pull request by robertwb**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @aaltay

File: sdks/python/apache_beam/runners/direct/direct_runner.py [hunk 1]
```diff
@@ -40,6 +40,14 @@ class DirectRunner(PipelineRunner):
   def __init__(self):
     self._cache = None
 
+  def apply_CombinePerKey(self, transform, pcoll):
+    from apache_beam.runners.direct.helper_transforms import LiftedCombinePerKey
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/direct/direct_runner.py]: Could this import moved to the top?

Comment (robertwb) [inline sdks/python/apache_beam/runners/direct/direct_runner.py]: No, for the same reason the imports below can't be moved to the top. Copied the TODO.

**Comment by robertwb** [comment]
Thanks. Merging.

**Pull request closed without merging**